app.js

```diff
@@ -88,8 +98,86 @@ function refreshCurrentGameEl() {
 
 function displayAllGames() {
     // clear out the past games list in the DOM
-
+    pastGamesEl.textContent = '';
     // loop through the past games in state
     // use the renderGame function to render and append a past game for each past game in state
     // again, review the renderGame function in render-utils.js. How many arguments does it take? What order does it take them in?
+    for (let games of pastGames) {
+        name1 = games.name1;
+        name2 = games.name2;
+        score1 = games.score1;
+        score2 = games.score2;
+        let gameHistoryDiv = renderGame(name1, name2, score1, score2);
```

Comment:
Totally works and feel free to ignore this: you can also do `const gameHistoryDiv = renderGame(games.name1, games.name2, ...etc)`

Another thing with naming conventions, you've declared `let games of pastGames` but what the variable games is is an individual game, either `game` or `pastGame` would perhaps be more accurate.